docs/workflows/_sdk-setup/_installation-react-native-expo.mdx

```diff
@@ -0,0 +1,62 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+```bash
+expo install @beyondidentity/bi-sdk-react-native
+```
+
+Add the SDK [config plugin](https://docs.expo.dev/guides/config-plugins/) to the [plugins array](https://docs.expo.dev/versions/latest/config/app/#plugins) of your app.{json,config.js,config.ts}:
+
+```json
+{
+  "expo": {
+    "plugins": [["@beyondidentity/bi-sdk-react-native"]]
+  }
+}
+```
+
+The SDK requires certain minimun native versions. You can set these requirments either with another plugin, [expo-build-properties](https://docs.expo.dev/versions/latest/sdk/build-properties/), or by modifing project [static files](https://docs.expo.dev/guides/config-plugins/#static-modification).
+
+<Tabs groupId="requirements" queryString>
+<TabItem value="expo-build-properties" label="expo-build-properties">
+
+```bash
+expo install expo-build-properties
+```
+
+```json
+{
+  "expo": {
+    "plugins": [
+      ["@beyondidentity/bi-sdk-react-native"],
+      [
+        "expo-build-properties",
+        {
+          "android": {
+            "minSdkVersion": 26
+          },
+          "ios": {
+            "deploymentTarget": "13.0"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+</TabItem>
+<TabItem value="static-files" label="static files">
+
+```bash title="android/gradle.properties"
+android.minSdkVersion=26
+```
+
+```bash title="ios/Podfile.properties.json"
+"ios.deploymentTarget": "13.0"
+```
+
+</TabItem>
+</Tabs>
+
+Finally, rebuild your app as described in Expo's [Adding custom native code guide](https://docs.expo.dev/workflow/customizing/).
```

Comment:
you link to this same article earlier in the guide, prior to installing the SDK. is that intentional?

Reply:
Yea because they need to set up their application following that guide first and finally they need to rebuild the application after installing our SDK.

Reply:
ok, right. i wasn't sure if there was a subtle difference, but it seems from what you are saying that the article is necessary both on the initial setup and on the rebuild. Thanks!